server/docs/finalGrades.yaml

```diff
@@ -131,3 +153,56 @@ definitions:
         $ref: '#/components/responses/Conflict'
     security:
       - cookieAuth: []
+
+/v1/courses/{courseId}/final-grades/{fGradeId}:
+  put:
+    tags: [FinalGrades]
+    description: >
+      Edit a final grade.
+      Available only to admin users and teachers in charge of the course.
```

Comment:
A general comment, do we need to specify that a feature is accessible to admin users, unless it is exclusively accessible to admin users? Admins should have access to everything anyway, so stating it in every endpoint is a bit redundant.